src/transformers/models/transfo_xl/modeling_transfo_xl.py

```diff
@@ -632,6 +633,40 @@ class TransfoXLModelOutput(ModelOutput):
     attentions: Optional[Tuple[torch.FloatTensor]] = None
 
 
+@dataclass
+class TransfoXLSequenceClassifierOutputWithPast(ModelOutput):
```

Comment:
This new class needs to be documented in the `.rst` with the other TransfoXL-specific outputs.